examples/ble_color_picker.py

```diff
@@ -17,7 +17,10 @@
         pass
 
     while uart_server.connected:
-        packet = Packet.from_stream(uart_server)
+        try:
+            packet = Packet.from_stream(uart_server)
+        except (ValueError, OSError):
+            pass
         if isinstance(packet, ColorPacket):
```

Comment:
Would it be better to put line 24-26 after line 21 in the successful try so they are no executed if the error is thrown?

Reply:
Also do you want to to "fail silently" Should the error be printed?

Reply:
Usually this fails because the connection has been broken after the `while` test for `connected`. So it would be best to fail completely and then go back and test for `connected` again.

Reply:
> Would it be better to put lines 24-26 after line 21 in the successful try so they are not executed if the error is thrown?

According to https://mail.python.org/pipermail/tutor/2005-July/040116.html you can use a try except continue inside a loop and it will go to the next iteration of the loop. I found this to be true in my fix as it would catch the bad pack and immediately grab a new packet without using hitting line 24 and erroring out.

Reply:
Is pass the same as continue in this case?

Reply:
> Is pass the same as continue in this case?

According to 
https://stackoverflow.com/questions/9483979/is-there-a-difference-between-continue-and-pass-in-a-for-loop-in-python  they are not.  I should have used continue in my PR as that would force the loop to start over.

Reply:
@dhalbert : Should I correct my PR to use continue, or do you just want me to close it and take a different approach?

Reply:
`continue` makes sense.

Reply:
> Usually this fails because the connection has been broken after the `while` test for `connected`. So it would be best to fail completely and then go back and test for `connected` again.

I was able to get the fail to happen after pressing many different buttons where it was obvious it couldn't keep up with the input. So in my case it should still have been connected..  but not sure how to test that..  hmmm I could perhaps in my except clause check the connection status.

Reply:
I changed my "fix" which is really just a workaround as @jerryneedell and I discussed in discord. I am now just catching the ValueError, printing out that I caught it and "continue"ing on to the next loop iteration.  I can really only get this error in more complicated animations where you can issue more commands from the phone app than you can process in your program. I still think it's helpful as an example in the code on what you can do as a work around if your packet generates an error and does no harm if it doesn't.